sources/packages/web/src/components/layouts/aest/AESTApplicationSideBar.vue

```diff
@@ -115,11 +142,25 @@ export default defineComponent({
           });
         }
       });
+      const applicationDetail =
+        await ApplicationService.shared.getApplicationDetail(
+          props.applicationId,
+        );
+      isDraftApplication.value =
+        applicationDetail.applicationStatus === ApplicationStatus.Draft;
+      const institutionLocationDetail =
+        await InstitutionService.shared.getInstitutionLocation(
```

Comment:
Do we need to make this 2nd api call if we are only checking if it is a `isBCPublicInstitution`?

Reply:
`isBCPublicInstitution` is removed.